documentation/docs/plugins/deploy.md

```diff
@@ -0,0 +1,40 @@
+---
+displayed_sidebar: pluginsSidebar
+---
+
+# Plugins Deployment
+
+Like said previously plugins must be hosted on your own infrastructure. This gives you the full control on the techonologies and when you want to update your plugin(s).
+
+## Where to deploy
+
+You can deploy your plugin / app anywhere. But it's import to make the plugin accessible publicly from the web. Other feature won't be available for customers and users.
```

Comment:
```suggestion
You can deploy your plugin/app anywhere. But it's import to make the plugin accessible publicly from the web. Other features won't be available for customers and users.
```

Reply:
The last sentence is not clear. What features are we talking about?

Reply:
Good catch, I'll update the sentence. `Otherwise, the features of your plugin will not be visible to users.`